Userland/Services/WebContent/WebDriverClient.ipc

```diff
@@ -58,5 +58,6 @@ endpoint WebDriverClient {
     take_screenshot() => (Web::WebDriver::Response response)
     take_element_screenshot(String element_id) => (Web::WebDriver::Response response)
     print_page() => (Web::WebDriver::Response response)
+    release_actions() => (Web::WebDriver::Response response)
```

Comment:
This is super duper nit picky, but I've grown to have a strong preference for keeping these in spec order in all files 🙃 

So in this spot, WebDriverConnection.h, and WebDriver/Client.h, `release_actions` should be between `delete_all_cookies` and `dismiss_alert`.